### ISSUE TITLE
Using `p/` in Name for adddriver throws error message

### ISSUE THREAD

**Issue by tomascherian** [tomascherian/pe#2]
**Steps to reproduce**:
* `adddriver p/98765432 n/Adam p/ayne`

![image.png](https://raw.githubusercontent.com/tomascherian/pe/main/files/3738c6f1-e449-4630-b1f1-397cd05ef2f3.png)

**Expected Output**: Add new driver with name `Adam p/ayne`

**Actual Output**: Error message for invalid phone number

**Comment by nus-pe-script** [comment]
# Team's Response
No details provided by team.

## The 'Original' Bug

[The team marked this bug as a duplicate of the following bug]

># Adding new customer with already existing phone number throws invalid phone number error message
>
>**Steps to reproduce**:
>* `addcustomer p/98765432 n/Adam p/ayne a/address`
>
>![image.png](https://raw.githubusercontent.com/nus-cs2103-AY2122S2/pe/master/files/a6184417-8baf-45d9-a5b9-2a974dc794e5.png)
>
>
>**Expected Output**: Already existing phone number error message is thrown
>
>
>**Actual Output**: Invalid phone error message is thrown
>
>
>
>
>
>
><hr>
><sub>[original: nus-cs2103-AY2122S2/pe-interim#3437] [original labels: type.FunctionalityBug severity.Medium]</sub>
>
>
>

## Their Response to the 'Original' Bug

[This is the team's response to the above 'original' bug]

>As stated in the UG, only the last occurrence of the parameter will be taken. In this case, `p/ayne` was taken to be the phone number field. 
>Since `ayne` is an invalid phone number, the error message is thrown. 
>
>![Screenshot 2022-04-18 at 3.54.04 PM.png](https://raw.githubusercontent.com/nus-cs2103-AY2122S2/pe/master/files/28ca3fb4-910a-4daf-8251-93e517597522.png)


# Items for the Tester to Verify
## :question: Issue duplicate status

Team chose to mark this issue as a duplicate of another issue (as explained in the _**Team's response**_ above)







- [x] I disagree

**Reason for disagreement:** The original issue is about an error message mismatch for `addcustomer`. This report is about restricting the use of symbols in `Name`

-------------------
## :question: Issue response

Team chose [`response.Rejected`]







- [x] I disagree

**Reason for disagreement:** Thank you for the response.

The use of of symbols in name throws a invalid name error message. Restricting the use of symbols cause user inconvenience.

Granted p/ayne would make the command hard to parse, but use of other symbols like - + # also cause this bug. e.g. adddriver p/98653200 n/Adam-payne also causes this bug.

-------------------
## :question: Issue type

Team chose [`type.FunctionalityBug`]
Originally [`type.FeatureFlaw`]

- [ ] I disagree

**Reason for disagreement:** [replace this with your explanation]

-------------------
